opal/mca/reachable/netlink/configure.m4

```diff
@@ -74,15 +76,17 @@ AC_DEFUN([OPAL_REACHABLE_NETLINK_CHECK_LIBNL3],[
        save_CPPFLAGS=$CPPFLAGS
        save_LIBS=$LIBS
 
-       $1="-I/usr/include/libnl3"
-       CPPFLAGS="$$1 $CPPFLAGS"
-       AC_MSG_CHECKING([for /usr/include/libnl3])
-       AS_IF([test -d "/usr/include/libnl3"],
-             [AC_MSG_RESULT([present])
-              AC_CHECK_HEADER(
-                [netlink/version.h],
-                [AC_COMPILE_IFELSE(
-                    [AC_LANG_PROGRAM([[
+       AS_IF([test $opal_libnl_version -eq 1],
```

Comment:
I think that much of the body of this macro can move to a new macro -- such as OPAL_LIBNL_CHECK_VERSION (i.e., these are the kinds of tests we want to use to figure out what version of libnl is available) -- and then both this macro and OPAL_CHECK_LIBNL -- simply calls OPAL_LIBNL_CHECK_VERSION to figure out what version of libnl is found.